src/gimelstudio/corenodes/filter/opacity_node/opacity_node.py

```diff
@@ -0,0 +1,85 @@
+# ----------------------------------------------------------------------------
+# Gimel Studio Copyright 2019-2021 by Noah Rahm and contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------
+
+
+from gimelstudio import api
+
+
+class OpacityNode(api.Node):
+    def __init__(self, nodegraph, _id):
+        api.Node.__init__(self, nodegraph, _id)
+
+    @property
+    def NodeMeta(self):
+        meta_info = {
+            "label": "Opacity",
+            "author": "Gimel Studio",
+            "version": (0, 0, 1),
+            "category": "FILTER",
+            "description": "",
+        }
+        return meta_info
+
+    def NodeInitProps(self):
+        self.value = api.PositiveIntegerProp(
+            idname="Opacity Value",
+            default=25,
+            min_val=0,
+            max_val=100,
+            widget=api.SLIDER_WIDGET,
+            label="Opacity:",
+        )
+        self.NodeAddProp(self.value)
+
+    def NodeInitParams(self):
+        image = api.RenderImageParam('Image')
+        self.NodeAddParam(image)
+
+    def NodeEvaluation(self, eval_info):
+        image1 = self.EvalParameter(eval_info, 'Image')
+        opacity_value = self.EvalProperty(eval_info, 'Opacity Value')
+
+        render_image = api.RenderImage()
```

Comment:
(Not the first time I've noticed this) Shouldn't we use a slightly less confusing name for the variables (maybe ``input_image`` instead of ``image1``?). Also consider using ``output_image`` (instead of ``render_image``)? I'm up for discussion on this. :slightly_smiling_face:

Reply:
Yes, I am going to open an issue specifically for this because Standards for the API is something I have been wanting to do for a while now! 😅 

Right now, different nodes use different variable names and it's kinda messy. I am open to suggestions so we can form a standard. It will also be important for the docs and for others to create custom nodes.